mm2src/mm2_main/tests/docker_tests/mod.rs

```diff
@@ -3,8 +3,8 @@ pub mod docker_tests_common;
 mod docker_ordermatch_tests;
 mod docker_tests_inner;
 pub mod qrc20_tests;
-mod slp_tests;
-mod solana_tests;
+#[cfg(not(feature = "disable-solana-tests"))] mod slp_tests;
```

Comment:
This is `SLP` (Simple Ledger Protocol) related tests, `slp` are BCH tokens. Solana tokens are `SPL` (Solana Program Library), I confuse the two of them alot too :)

Reply:
Ah so I confuse them again.. We should use long version of names for these. Like `solana_program_library` instead of `spl` and `simple_ledger_protocol` instead of `slp`.

good catch btw!

Reply:
> We should use long version of names for these

Agree.